bin/CodeChecker.py

```diff
@@ -193,6 +193,21 @@ def signal_handler(sig, frame):
 
         subparsers = parser.add_subparsers(help='commands')
 
+        # TODO: Delete these once v6.0 rolls.
+        old_subcommands = []
+
+        def _warn_deprecated_command(cmd_name):
+            # Write to stderr so the output is not captured by pipes, e.g.
+            # with the "checkers" command, the "-" in the new command's name
+            # would mess up pipe usage.
+            sys.stderr.write(
+                "[WARNING] The called command 'CodeChecker {0}' is DEPRECATED "
```

Comment:
I do not think we should warn the users before v6.0 is released. After the v6.0 release we should inform them that a new option is available and the old might be removed in one of the later releases.